src/Libt8.jl

```diff
@@ -1,6 +1,6 @@
 module Libt8
 
-using CEnum: CEnum, @cenum
+using CEnum
```

Comment:
```suggestion
using CEnum:@cenum
```
Hm, this line is [written by Clang.jl](https://github.com/JuliaInterop/Clang.jl/blob/0a0d245be658b814c498ed4ac85578770d692dc8/src/generator/passes.jl#L1122). So, the explicit imports cannot be enforced by us. Maybe it would be nice to include the explicit imports in Clang.jl, see my PR https://github.com/JuliaInterop/Clang.jl/pull/488 over there.

Reply:
We do not actually use `CEnum` explicitly. So we only need `@cenum`.

Reply:
Alternatively, we could put this in the [fixes.sh](https://github.com/DLR-AMR/T8code.jl/blob/main/dev/fixes.sh).